### PR TITLE
Switched data storage to use RDBMS instead of redis

### PR DESCRIPTION
**Pull request by jackofseattle**
to easy development we're now using sqlite3 for everything in development.

You no longer need to have redis installed or running on your local machine.
Redis is still used for sessions in production as it will keep down on the database size.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,18 +10,12 @@
...
 
-##Redis
-
-For development purposes, install [Docker Toolbox](https://www.docker.com/docker-toolbox), fire up Kitematic and install redis container through it to get a working Redis server with nearly zero config.
-Set the following env variable for your Redis DB host and port.
-
-```export REDIS_PORT=32768```
-
-```export REDIS_HOST=192.168.99.100```
+Updating the
```
Comment (jackofseattle) [inline README.md]: extra text - guess I forgot to finish my thought. Will remove.

**Comment by jimitpatel9** [comment]
:+1:

**Comment by prashantpawar** [comment]
:+1:

File: server/session.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+let sessionConf = {
+  secret: process.env.SESSION_SECRET || 'get smarter',
+  resave: false,
+  saveUninitialized: false
+};
+
+
+if (config.redis_host) {
```
Comment (prashantpawar) [inline server/session.js]: Actually I am curious, why are we keeping redis(and using it for session management)?

Comment (jackofseattle) [inline server/session.js]: No real reason other than db size. It costs us nothing in the long run really.

**Pull request merged**